src/renderer/components/WebPlatformPlayer/TopBar.js

```diff
@@ -123,9 +128,14 @@ const WebPlatformTopBar = ({ manifest, onReload, onHelp, onClose }: Props) => {
         </ItemContent>
       </ItemContainer>
       <RightContainer>
-        {/* <ItemContainer isInteractive onClick={onHelp}>
+        <ItemContainer
+          isInteractive
+          onClick={() => {
```

Comment:
try using as much as possible `useCallback` when you can for inline callbacks to minimize reassigning functions on each render